packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -43,16 +41,6 @@ export const TabsItem = compose<TabsItemType>({
       }
     }, []);
 
-    // Ensure focus is placed on button after click
-    const changeSelectionWithFocus = useOnPressWithFocus(componentRef, changeSelection);
-
-    /* RadioButton changes selection when focus is moved between each RadioButton and on a click */
-    const pressable = useAsPressable({
-      ...rest,
-      onPress: changeSelectionWithFocus,
-      onFocus: changeSelection,
-    });
-
```

Comment:
In develop, removing this breaks the button, does the button still work without this? (try with alerts)

Reply:
When moving from pressable to button we can just use the buttons onClick property. I tested if the button clicking works when overriding the onClick property and it worked.

Reply:
Is the button also clicked when you select it (without pressing)? As it is Tabs, the button must be selected with the arrows, does that still work?

Reply:
Made a new commit that shows onFocus being overloaded. We can use that prop as an entry to handle selecting the tab onFocus.